remove experimental child aggregates (#755)
patchlevel/event-sourcing: docs/pages/aggregate.md

In some cases, it makes sense to split an aggregate into several smaller aggregates.
This can be the case if the aggregate becomes too large or if the aggregate is used in different contexts.

> We currently support only one pattern for this: Micro Aggregates.

Reviewer: I would rephrase this, as it contextually now not much sense to talk about only one pattern.

```suggestion
For these cases you can use Micro Aggregates.
```
